common/templates/emails/new_enrollment_email_body.txt

```diff
@@ -0,0 +1,5 @@
+Enrollment received for:
+learner: ${user.username} (${user.get_full_name()})
```

Comment:
When `get_full_name()` returns empty string it gives a bit strange effect, e.g. `tomasz ()`, so maybe brackets could be conditional?